services/EventTokenService.js

```diff
@@ -0,0 +1,67 @@
+const jwt = require("jsonwebtoken");
+const uuid4 = require("uuid4");
```

Comment:
Are we sure to use this package, as it does not always generate a unique token.
![image](https://github.com/Real-Dev-Squad/website-backend/assets/57758447/0f95a056-49f0-4687-a42c-067692109eb1)
picked it up from : https://www.npmjs.com/package/uuid

Reply:
Hey @RitikJaiswal75,

Thank you for letting me know. I have replaced this package with the `crypto` package to generate the unique IDs.